processor/spanmetricsprocessor/processor_test.go

```diff
@@ -493,13 +559,27 @@ func newOTLPExporters(t *testing.T) (*otlpexporter.Config, component.MetricsExpo
 func TestBuildKey(t *testing.T) {
 	span0 := pdata.NewSpan()
 	span0.SetName("c")
-	k0 := buildKey("ab", span0, nil)
+	k0 := buildKey("ab", span0, nil, false)
 
 	span1 := pdata.NewSpan()
 	span1.SetName("bc")
-	k1 := buildKey("a", span1, nil)
+	k1 := buildKey("a", span1, nil, false)
 
 	assert.NotEqual(t, k0, k1)
+
+	span2 := pdata.NewSpan()
+	span2.SetName("cd")
+	spanID := pdata.NewSpanID([8]byte{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02})
```

Comment:
If you want to make this slightly more readable, you can use `hex.DecodeString(s)`. Not a big deal if you're ceebs

Reply:
I'll just keep it like this if you don't feel strongly about it as it takes up less lines rather than having to assign it to variable and do error checking and etc.